docs/guides/platforms/windows/host-avalonia-controls-in-winforms.md

```diff
@@ -0,0 +1,56 @@
+---
+id: host-avalonia-controls-in-winforms
+title: Host Avalonia controls in Windows Forms
+---
+
+# Host Avalonia controls in Windows Forms
+
+Avalonia controls can be hosted in Windows Forms applications. This enables a step-by-step migration of existing Windows Forms applications to Avalonia.
+
+## Overview
+
+An exemplary Windows Forms application with Avalonia controls requires at least three projects:
+
+1. `YourApp` Cross platform library where you put your Avalonia controls 
+2. `YourApp.Desktop` Executable Avalonia application, required for the Visual Studio Avalonia designer
+3. `YourApp.WinForms` Your existing Windows Forms application
+
+## Prerequisites
+
+- Visual Studio 2022
+- Avalonia extension for Visual Studio 2022
+- A solution with a Windows Forms project
+
+## Step-by-step instructions
+
+1. Create both Avalonia projects
+   - In Visual Studio add a new project to your solution
+   - Choose _Avalonia C# Project_
+   - Select at least _Desktop_ as target platforms
+   - Click _Create_
+   - You should now have a `YourApp` and a `YourApp.Desktop` project in your solution
```

Comment:
Since it's about embedding Avalonia controls into a WinForms project. Do we really need `.Desktop` avalonia executable? We can keep it, but it should be clarified as optional. That `.Desktop` executable isn't really needed for WinForms.

Reply:
It's only required for the Visual Studio extension because that does not allow you to use the `YourApp` library project as hosting project and the WinForms project does not work as it does not start an Avalonia application.

I changed the description to make clearer that it's only required for Visual Studio.

Reply:
Makes sense. Thank you.